webui/react/src/pages/F_ExpList/glide-table/contextMenu.tsx

```diff
@@ -0,0 +1,223 @@
+import { ExclamationCircleOutlined } from '@ant-design/icons';
```

Comment:
it looks like this is mostly the same as `ExperimentActionDropdown.tsx`, can we just reuse that (we get the styling that way too).

the main challenge as far as i can tell is the positioning: `ExperimentActionDropdown.tsx` uses antd `Dropdown`, which  gets positioned relative to the triggering DOM element.

but we can override that by specifying `align={{ offset: [x, y] }}` and `getPopupContainer={/* a function that returns the div containing the table */}` in the `Dropdown` props.

Reply:
RE:

> antd.Table was triggering and filling the context menu dropdown in a way which isn't super accessible here

the `antd.Table` isn't doing anything with the dropdown, all the functionality is just from wrapping the individual rows in `antd.Dropdown`

Reply:
on second thought, I guess we might also want to add the ability to copy cells or open links in a new tab so maybe it makes sense for it to be it's own thing.